src/dswx_sar/mosaic_gcov_frame.py

```diff
@@ -1012,8 +1014,14 @@ def run(cfg):
     mosaic_cfg = processing_cfg.mosaic
     mosaic_mode = mosaic_cfg.mosaic_mode
     mosaic_prefix = mosaic_cfg.mosaic_prefix
+    nisar_uni_mode = processing_cfg.nisar_uni_mode
+
+    # Determine if resampling is required
```

Comment:
@hb6688 I hope you add more descriptions. `nisar_uni_mode` and `resamp_required`, and `resamp_out_res` are computed from `dswx_ni_runconfig.py`, not from user. I think this should be clarified and mentioned here.